### ISSUE TITLE
[Feature Request] "Alpha Edges" a secondary UV mapping for Edges

### ISSUE THREAD

**Issue by Corruptinator** [leukbaars/DreamUV#8]
Having an interesting food for thought: What if DreamUV utilized a **separate texture that uses an alpha** so then it can be mapped on a separate material and UV to wrap details such as edge wear, etc on the edges?

I'm sure you're aware of this but figure it may be something to point out mostly cause of round edges:
![image](https://user-images.githubusercontent.com/5027750/86636283-f7149480-bf88-11ea-8b45-73c53563063f.png)

The Circle UV island fits perfectly but leaves part of the UV Edge Wear out. So when I meant a separate texture with alpha for edges, I was thinking of this:

(Original):
![image](https://user-images.githubusercontent.com/5027750/86636598-5672a480-bf89-11ea-84ac-c224d98e3e20.png)

(Alpha Texture for Edges):
![image](https://user-images.githubusercontent.com/5027750/86636841-9e91c700-bf89-11ea-99ab-eaefad5517da.png)
_Not the best texture example, but nonetheless..._

So then using the alpha texture, DreamUV could calculate the edges where to bake the edges on an **another material** alongside the original _atlas texture_ material. While also utilizing a **secondary UV map**:

![image](https://user-images.githubusercontent.com/5027750/86637549-95edc080-bf8a-11ea-9f88-6016ad65fafb.png)
![image](https://user-images.githubusercontent.com/5027750/86637585-a69e3680-bf8a-11ea-8b59-eacf53291964.png)

I do imagine that the theoretical idea sounds difficult to implement but figure I mention this idea, just for curiousity. 😄

**Comment by leukbaars** [comment]
Yeah I think this idea is great but finding an implementation that is good for general use is a bit difficult.  Personally I prefer using decal strips to do this kind of stuff. Decal strips could potentially be generated from selected edges using a different script, which I dont feel like should be part of this toolset.
